index.js

```diff
@@ -0,0 +1 @@
+console.log("Hello, world!");
```

Comment:
I think we can remove this file since we can compile our ts files into js when we need it. We don't want to leave a trivial page that we might forget to clean up later